### ISSUE TITLE
Entity icons use an absolute path instead of a relative one

### ISSUE THREAD

**Issue by deepnight** [Ogmo-Editor-3/OgmoEditor3-CE#87]
```
"texture": "C:\\Projects\\secretPrototype\\res\\map\\icons\\helmet_017.png",
```
instead of something like:
```
"texture": "icons\\helmet_017.png",
```

**Comment by deepnight** [comment]
BTW: even if I manually edit the ``.ogmo`` file, it doesn't accept relative paths.

**Comment by AustinEast** [comment]
That makes sense. Also then it would match the tileset's relative path... this shouldn't be too hard to get in :)

**Comment by AustinEast** [comment]
fixed with commit 709bac15177bbaa606ad5c54b166a81c3a5b304a

I made sure this works with pre-existing Ogmo projects - it should "auto-upgrade" any project it opens that still has absolute paths for entity textures 👍
